Dataplex/tests/Unit/V1/ContentServiceClientTest.php

```diff
@@ -609,6 +609,196 @@ public function updateContentExceptionTest()
         $this->assertTrue($transport->isExhausted());
     }
 
+    /** @test */
+    public function getIamPolicyTest()
```

Comment:
for some reason this is writing the tests an extra time. We need to investigate.

cc @noahdietz

Reply:
This will be fixed in https://github.com/googleapis/gapic-generator-php/pull/584